checkpoint/cp-resource-management/templates/addons/image-uploader/role.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if .Values.rbac.pspEnable }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Release.Name }}-image-uploader
+  namespace: {{ .Release.Namespace }}

```

Comment:
in deployment and others you don't state namespace in YAML while here you do - is it correct?